cypress/utils/helpers.js

```diff
@@ -1,10 +1,14 @@
-import Chance from 'chance';
 
-const chance = new Chance();
+export function pickRandomProducts(allProducts, quantity = 1) {
+  return Cypress._.sampleSize(allProducts, quantity);
+}
 
-export function pickRandomProduct(listOfProducts) {
-  const selectedCategory = chance.pickone(listOfProducts.collections);
-  const selectedItem = chance.pickone(selectedCategory.items);
-  selectedItem.linkUrl = selectedCategory.linkUrl;
-  return selectedItem;
+export function pickTargetProducts(productsList, preview = false) {
```

Comment:
please check comment related to rework of adding products to cart method and make this helper to work with target collection, but not with whole products scope